integration/base/src/main/java/com/inrupt/client/integration/base/AuthenticationScenarios.java

```diff
@@ -125,23 +121,31 @@ static void setup() {
             podUrl += Utils.FOLDER_SEPARATOR;
         }
 
-        createAuthenticatedClient();
+        final Session session = OpenIdSession.ofClientCredentials(
+                URI.create(issuer), //Client credentials
+                CLIENT_ID,
+                CLIENT_SECRET,
+                AUTH_METHOD);
+
+        localAuthClient = SolidSyncClient.getClient().session(session);
+
         if (PUBLIC_RESOURCE_PATH.isEmpty()) {
             publicTestContainerURI = URIBuilder.newBuilder(URI.create(podUrl))
-                    .path("test-" + UUID.randomUUID())
+                    .path("auth-test-" + UUID.randomUUID())
                     .build();
 
         } else {
             publicTestContainerURI = URIBuilder.newBuilder(URI.create(podUrl))
                 .path(PUBLIC_RESOURCE_PATH)
-                .path("test-" + UUID.randomUUID())
+                .path("auth-test-" + UUID.randomUUID())
                 .build();
 
             publicContainerURI = URIBuilder.newBuilder(URI.create(podUrl))
                     .path(PUBLIC_RESOURCE_PATH + "/").build();
 
-            createContainer(publicContainerURI);
-            prepareACR(publicContainerURI);
+            //if a tests fails it can be that the cleanup was not properly done, so we do it here too
+            Utils.deleteContentsRecursively(localAuthClient, publicContainerURI);
```

Comment:
As mentioned elsewhere, we can't delete this data here, because it will interfere with other concurrently-running tests.